source/_integrations/knocki.markdown

```
@@ -16,12 +16,32 @@ ha_platforms:
+7. Follow the in-app instructions to add the Home Assistant task.
 
 {% include integrations/config_flow.md %}
+
+## Automation example
+1. Go to {% my automations title="**Settings** > **Automations & scenes**" %} and in the lower right corner, select  **Create Automation**.
+2. Select **Create new automation**.
+3. Select **Add Trigger**, select **Entity** as the trigger type.  
+   - Search for the name of the task (as you named it in the Knocki app). Select the entity with that name appended to it.  
+   - Select **State** as the type of entity change.
+4. Select **Add action** and choose whatever action(s) you’d like the automation to trigger.
+5. Finally, test the action by tapping the associated gesture pattern. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)